master/docs/developer/cls-forcesched.rst

```diff
@@ -71,9 +67,15 @@ This section contains information to help users or developers who are interested
 
     .. py:attribute:: type
 
-           A list of types that the parameter conforms to.
-           These are used by the jinja template to create appropriate html form widget.
-           The available values are visible in ``master/buildbot/status/web/template/forms.html`` in the ``force_build_one_scheduler`` macro.
+           A string identifying the type that the parameter conforms to.
+           It is used by the angular application to find which angular directive to use for showing the form widget.
+           The available values are visible in ``www/base/src/app/common/directives/forcefields/forcefields.directive.coffee``.
```

Comment:
I think link on the source file will be better:

```
The available values are visible in :src:`www/base/src/app/common/directives/forcefields/forcefields.directive.coffee`.
```

Previously I removed link because `forms.html` was not present in the source tree and Sphinx' linkcheck warned about it.